torchft/local_sgd.py

```diff
@@ -176,19 +176,27 @@ def __init__(
         use_bucketization: bool = False,
         bucket_cap_mb: Optional[int] = None,
         should_quantize: bool = False,
+        fragment_sync_delay: int = 0,
+        fragment_update_alpha: float = 0.0,
     ) -> None:
         self._manager = manager
         self._model_fragment = model_fragment
         self._fragment_sync_offset = fragment_sync_offset
         self._local_optimizer = inner_optimizer
-        self._local_step = 0
         self._sync_every = sync_every
         assert sync_every >= 1, "sync_every must be greater than or equal to 1"
         self._backup_device = backup_device
         self._pin_memory = pin_memory
+        self._fragment_sync_delay = fragment_sync_delay
```

Comment:
probably need some validation logic to check relationships between  `fragment_sync_delay` , `fragment_sync_offset` and `sync_every`

e.g. `fragment_sync_delay` < `fragment_sync_offset`, `fragment_sync_offset` < `sync_every`, etc.